flutter/lib/src/sentry_flutter.dart

```diff
@@ -34,6 +34,11 @@ mixin SentryFlutter {
     if (optionsConfiguration == null) {
       throw ArgumentError('OptionsConfiguration is required.');
     }
+
+    // It is necessary to initialize Flutter method channels so that
+    // our plugin can call into the native code.
+    WidgetsFlutterBinding.ensureInitialized();
```

Comment:
I'm not against the change, but I also don't see why, there's a difference between adding the default integrations to the list and executing them.

https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/sentry.dart#L111-L114

the integrations only get the chance to run after the SDK is fully initialized, which is the last step, at this point, the SDK is enabled and `WidgetsFlutterBinding.ensureInitialized` has been called for quite a bit.

Reply:
Ah I see. I was not aware of that.  Still, I would argue for `WidgetsFlutterBinding.ensureInitialized()` because it seems to me this is a better practice. However I'm fine with either solution.

Reply:
let's merge as it is, thanks :)